package.json

```diff
@@ -1,29 +1,30 @@
 {
   "name": "opendistro_index_management_kibana",
-  "version": "1.11.0.0",
+  "version": "1.12.0.0",
   "description": "Kibana plugin for Index Management",
   "main": "index.js",
   "license": "Apache-2.0",
   "homepage": "https://github.com/opendistro-for-elasticsearch/index-management-kibana-plugin",
   "kibana": {
-    "version": "7.9.1",
+    "version": "7.10.0",
     "templateVersion": "1.0.0"
   },
```

Comment:
Minor: this can be removed, since we maintain the Kibana version in the `kibana.json` manifest file now.

Reply:
Removed